rash_core/src/context.rs

```diff
@@ -31,7 +31,8 @@ impl Context {
         Context { tasks, vars }
     }
 
-    /// Execute first [`task::Task`] and return a new context without that executed [`task::Task`]
+    /// Execute the first [`task::Task`] and return a new context without the one
+    /// that executed [`task::Task`]
```

Comment:
Our line width limit is 120. Again I dunno if I'm consistent in all the code but I would like.

Reply:
Cool, I didn't know that. Would you consider adding a .editorconfig to the repo so the code style policy is public? That would be really helpful to newcomers like me 😁

Reply:
mmmm... Ok, I was wrong it is [100](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#max_width), we are using default rustfmt values.

Reply:
Oops, the line is too long now then... But it's not a big deal, I say we clean house when we have an `.editorconfig` file!